### PR TITLE
Update getting started to include MacOS

### PR DESCRIPTION
**Pull request by Cmdv**
Adding more information in the "getting started" documentation to reflect what is present on https://developers.cardano.org/docs/get-started/installing-cardano-node/#macos

### CODE REVIEW

**Comment by angerman** [comment]
This looks good to me; and reminds me we have a an even less involved solution for on systems where nix is installed; which I should document better. For those that have nix installed or are ok to install it, and don't want to deal with `brew` or similar.

```
$ nix develop github:input-output-hk/devx#ghc8107 --refresh --no-write-lock-file
...
 _____ _____ _____    _____         _       _ _    _____ _       _ _ 
|     |     |   __|  |  |  |___ ___| |_ ___| | |  |   __| |_ ___| | |
|-   -|  |  |  |  |  |     | .'|_ -| '_| -_| | |  |__   |   | -_| | |
|_____|_____|_____|  |__|__|__,|___|_,_|___|_|_|  |_____|_|_|___|_|_|
                                                                     
[~/Projects/iohk/cardano-node]$ export DYLD_LIBRARY_PATH=$(pkg-config --libs-only-L libcrypto|cut -c 3-) # this one is only needed on macOS right now, due to a bug in loading libcrypto.
[~/Projects/iohk/cardano-node]$ cabal update; cabal build cardano-cli cardano-node -j --ghc-options=-j
...
```

which works on intel, and apple silicon Macs, as well as linux, and WSL the same. (Takes about ~40min on a m2 air)

**Comment by Cmdv** [comment]
@angerman think I might have to give the nix version a go as I'm getting LLVM version recognition errors 😞

**Comment by angerman** [comment]
> @angerman think I might have to give the nix version a go as I'm getting LLVM version recognition errors 😞
> 
> edit: I got passed that and then openSSL came to bit me.

Yea, that's why input-output-hk/devx now exists. We are working on a faster bootstrap, and might at some point even try to get something that works without nix installed. For now it requires nix to be installed first :-/

**Comment by Cmdv** [comment]
> Yea, that's why input-output-hk/devx now exists. We are working on a faster bootstrap, and might at some point even try to get something that works without nix installed. For now it requires nix to be installed first :-/

Interesting good to know 👍

File: doc/getting-started/install.md [hunk 1]
```diff
@@ -47,7 +47,26 @@ sudo apt-get install automake build-essential pkg-config libffi-dev libgmp-dev l
...
+Then using homebrew install the following:
+
+```bash
+brew install jq libtool autoconf automake pkg-config openssl
+```
+#### You will need to install llvm in case you are using M1
+
+```
+brew install llvm@13
+```
```
Comment (angerman) [inline doc/getting-started/install.md:69]: This we won't need once we are on 9.2 🎉

**Review by angerman — APPROVED** [review]

**Comment by angerman** [comment]
@olgahryniuk @CarlosLopezDeLara, you ok with this? Can this be merged?

**Comment by angerman** [comment]
@Cmdv this will need to be rebased onto master to get the status checked green.

**Comment by Cmdv** [comment]
cool let me do that now 😄

**Comment by angerman** [comment]
@Cmdv you'll also need to sign all the commits
> The base branch requires all commits to be signed. [Learn more about signing commits.](https://docs.github.com/articles/about-gpg/)

**Comment by Cmdv** [comment]
grrrrr my emacs GPG has stopped working!! going to sign the previous commits manually!!

**Review by newhoggy — APPROVED** [review]

**Comment by angerman** [comment]
@newhoggy why doesn't this trigger any GHAs?

**Comment by newhoggy** [comment]
I believe it's because it is from a fork which doesn't run an actions for security reasons.

**Comment by newhoggy** [comment]
I triggered it manually.

**Comment by angerman** [comment]
> I triggered it manually.

Don't we usually get some "this is from a fork, do you want to run the actions" button?

**Pull request merged**